src/backend/startup.js

```diff
@@ -0,0 +1,16 @@
+import { firestore } from '../config/firebase';
+
+//Default Room Name can change this later
+var roomName = "vip";
+var playerID = "";
+
+export async function register(name) {
+    await firestore.collection(roomName).add({
+        name: name,
+    })
+    .then(function(docRef) {
+        playerID = docRef.id;
+        console.log("Document written with ID: ", playerID);
+    })
```

Comment:
Since you defined a .then is this function still async?

Reply:
I think `register` is meant to be an exported async function that's made up of async + sync(callback) logic.